<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>254</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_Random">
      <property name="text">
       <string>Random</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_Bet">
      <property name="text">
       <string>Bet</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_Down">
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_Raise">
      <property name="text">
       <string>Raise</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_commit">
      <property name="text">
       <string>Commit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_bet">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_coin">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_random">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>SizeHorCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_beting">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Betting coin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_OppentCard">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Oppent Card</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Coin">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Your coin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>181</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="mypro.qrc">:/new/prefix1/player.PNG</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="mypro.qrc">:/new/prefix1/title.PNG</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>lcdNumber_bet</zorder>
  <zorder>lcdNumber_coin</zorder>
  <zorder>label_beting</zorder>
  <zorder>label_OppentCard</zorder>
  <zorder>label_Coin</zorder>
  <zorder>label_2</zorder>
  <zorder>lcdNumber_random</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mypro.qrc"/>
 </resources>
 <connections/>
</ui>
